### PR TITLE
Add multipage business cards test form recognizer

### PR DESCRIPTION
**Pull request by samvaity**
- Add tests to cover scenarios for multipage documents with one business card per page for business card recognition API in Form Recognizer.
- Update test methods to combine receipt and business card testing methods.

### CODE REVIEW

File: sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTest.java [hunk 1]
```diff
@@ -1528,10 +1551,30 @@ public void recognizeBusinessCardSourceUrlWithPngFile(HttpClient httpClient,
...
+     */
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.formrecognizer.TestUtils#getTestParameters")
+    public void recognizeMultipageBusinessCardUrl(HttpClient httpClient, FormRecognizerServiceVersion serviceVersion) {
+        client = getFormRecognizerClient(httpClient, serviceVersion);
+        urlRunner(sourceUrl -> {
+            SyncPoller<FormRecognizerOperationResult, List<RecognizedForm>> syncPoller =
+                client.beginRecognizeBusinessCardsFromUrl(sourceUrl,
+                    new RecognizeBusinessCardsOptions()
+                        .setContentType(FormContentType.APPLICATION_PDF)
```
Comment (kristapratico) [inline sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTest.java:1569]: if recognize from url, shouldn't content-type be application/json?

**Review by kristapratico — APPROVED** [review]

File: sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTest.java (hunk 1 above)
Comment (samvaity) [reply sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTest.java:1569]: yes, this value is ignored, will update!

File: sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java [hunk 2]
```diff
@@ -799,10 +777,32 @@ static void validateMultiPageDataUnlabeled(List<RecognizedForm> actualRecognized
...
             });
         });
     }
+    static void validateMultipageBusinessData(List<RecognizedForm> recognizedBusinessCards) {
+        assertEquals(2, recognizedBusinessCards.size());
+        RecognizedForm businessCard1 = recognizedBusinessCards.get(0);
+        RecognizedForm businessCard2 = recognizedBusinessCards.get(1);
+
+        assertEquals(1, businessCard1.getPageRange().getFirstPageNumber());
+        assertEquals(1, businessCard1.getPageRange().getLastPageNumber());
```
Comment (mssfang) [inline sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java:789]: first page number is same as last page number?

File: sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java [hunk 3]
```diff
@@ -655,29 +627,35 @@ void validateRecognizedResult(List<RecognizedForm> actualFormList, boolean inclu
...
+                    assertEquals(expectedReceiptFields.get("ReceiptType").getConfidence(),
+                        actualRecognizedReceiptFields.get("ReceiptType").getConfidence());
+                    validateFieldValueTransforms(rawDocumentResult.getFields().get(receiptField),
+                        actualRecognizedReceiptFields.get(receiptField), rawReadResults, includeFieldElements);
+                });
+            }
         }
     }
 
+
```
Comment (mssfang) [inline sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java:658]: nit: extra line

**Review by mssfang — APPROVED** [review]
LGTM!

File: sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java [hunk 4]
```diff
@@ -110,6 +115,14 @@
     static final List<String> BUSINESS_CARD_FIELDS = Arrays.asList("ContactNames", "JobTitles", "Departments",
         "Emails", "Websites", "MobilePhones", "OtherPhones", "Faxes", "Addresses", "CompanyNames");
 
+    // Business Card fields
```
Comment (maririos) [inline sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java:118]: nit: wrong comment

**Review by maririos — APPROVED** [review]

File: sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java [hunk 5]
```diff
@@ -799,10 +777,32 @@ static void validateMultiPageDataUnlabeled(List<RecognizedForm> actualRecognized
...
+        Map<String, FormField> businessCard1Fields = businessCard1.getFields();
+        List<FormField> emailList = businessCard1Fields.get("Emails").getValue().asList();
+        assertEquals("johnsinger@contoso.com", emailList.get(0).getValue().asString());
+        List<FormField> phoneNumberList = businessCard1Fields.get("OtherPhones").getValue().asList();
+        assertEquals("+14257793479", phoneNumberList.get(0).getValue().asPhoneNumber());
+        assertNotNull(businessCard1.getPages());
+
+        assertEquals(2, businessCard2.getPageRange().getFirstPageNumber());
+        assertEquals(2, businessCard2.getPageRange().getLastPageNumber());
+        Map<String, FormField> businessCard2Fields = businessCard2.getFields();
```
[the remark this answers is not in the document]
Comment (samvaity) [reply sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java:799]: I don't have the contact name page number changes in yet, but yes will check here once I do.

**Pull request merged**